fix "set_webapp_settings"

Mapel88 opened the pull request:
Removes 'r' prefix from this section cause the command is not working with it...
 if setting == "userName" or setting == "password":
            pscmd.append(r" Set-WebConfigurationProperty -Filter \"system.applicationHost/sites/site[@name='{0}']/application[@path='/{1}']/virtualDirectory[@path='/']\"".format(site, name))
            pscmd.append(r" -Name \"{0}\" -Value {1};".format(setting, value))

        if setting == "physicalPath" or setting == "applicationPool":
            pscmd.append(r" Set-ItemProperty \"IIS:\Sites\{0}\{1}\" -Name {2} -Value {3};".format(site, name, setting, value))

### What does this PR do?

### What issues does this PR fix or reference?

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Tests written?

Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

Mapel88 committed:
fix "set_webapp_settings"

Removes 'r' prefix from this section cause the command is not working with it...
 if setting == "userName" or setting == "password":
            pscmd.append(r" Set-WebConfigurationProperty -Filter \"system.applicationHost/sites/site[@name='{0}']/application[@path='/{1}']/virtualDirectory[@path='/']\"".format(site, name))
            pscmd.append(r" -Name \"{0}\" -Value {1};".format(setting, value))

        if setting == "physicalPath" or setting == "applicationPool":
            pscmd.append(r" Set-ItemProperty \"IIS:\Sites\{0}\{1}\" -Name {2} -Value {3};".format(site, name, setting, value))

salt-jenkins commented:
@Mapel88, thanks for your PR! By analyzing the history of the files in this pull request, we identified @twangboy, @m03 and @trebortech to be potential reviewers.

cachedout reviewed (changes requested):
Please fix up these lint errors before we merge this: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/10156/violations/file/salt/modules/win_iis.py/

Mapel88 committed:
Fixed standardization issues in win_iis

- Fixed lint error in set_webapp_settings
- Added SaltInvocationError to data input checks
- Fixed some standardization in set_webapp_settings & get_webapp_settings

Commit:
[Lint] Fix spaces around variable assignment
